### PR TITLE
OTT-197: Log Partner bidder code in video event tracker

### PR DESCRIPTION
**Pull request by Pubmatic-Dhruv-Sonone**
OTT-197: Log Partner bidder code in video event tracker

### CODE REVIEW

**Comment by ShriprasadM** [comment]
@Pubmatic-Dhruv-Sonone 

I can see changes are made in `requestBid` function. It is a core part of the PBS framework.
`bidderCoreName` is introduced here.

As we have discussed, when will raise PR for contributing this change, we may need to justify why we have made changes in `requestBid` function.

Can you check, if any other way is available with minimum changes? I am also looking into it

**Comment by PubMatic-OpenWrap** [comment]
> @Pubmatic-Dhruv-Sonone
> 
> I can see changes are made in `requestBid` function. It is a core part of the PBS framework.
> `bidderCoreName` is introduced here.
> 
> As we have discussed, when will raise PR for contributing this change, we may need to justify why we have made changes in `requestBid` function.
> 
> Can you check, if any other way is available with minimum changes? I am also looking into it

@ShriprasadM  Made the changes as per our discussion

**Review by ShriprasadM — APPROVED** [review]
Address the rename variable change.
Otherwise LGTM.

File: endpoints/events/vtrack.go [hunk 1]
```diff
@@ -344,7 +346,7 @@ func ModifyVastXmlJSON(externalUrl string, data json.RawMessage, bidid, bidder,
 
 //InjectVideoEventTrackers injects the video tracking events
 //Returns VAST xml contains as first argument. Second argument indicates whether the trackers are injected and last argument indicates if there is any error in injecting the trackers
-func InjectVideoEventTrackers(trackerURL, vastXML string, bid *openrtb2.Bid, bidder, accountID string, timestamp int64, bidRequest *openrtb2.BidRequest) ([]byte, bool, error) {
+func InjectVideoEventTrackers(trackerURL, vastXML string, bid *openrtb2.Bid, bidder, bidderCoreName, accountID string, timestamp int64, bidRequest *openrtb2.BidRequest) ([]byte, bool, error) {
```
[the remark this answers is not in the document]
Comment (PubMatic-OpenWrap) [reply endpoints/events/vtrack.go]: Done

**Pull request merged**